XCUITests/NavigationTest.swift

```diff
@@ -9,7 +9,7 @@ let website_2 = ["url": "www.example.com", "label": "Example", "value": "example
 
 let urlAddons = "addons.mozilla.org"
 let urlGoogle = "www.google.com"
-let popUpTestUrl = "http://www.popuptest.com/popuptest1.html"
+let popUpTestUrl = path(forTestPage: "test-popup-blocker.html")
```

Comment:
is this a new way to open a local html file?

Reply:
Well, it is not new, we have been using that for a while now :) but for this test we were using an external site not a local one